pygmt/src/inset.py

```diff
@@ -0,0 +1,131 @@
+"""
+inset - Create inset figures.
+"""
+import contextlib
+
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@contextlib.contextmanager
+@use_alias(D="position", F="box", M="margin", N="no_clip", V="verbose")
+@kwargs_to_strings(D="sequence", M="sequence")
+def inset(self, **kwargs):
+    r"""
+    Create an inset within a figure to plot an additional figure.
```

Comment:
```suggestion
    Create an inset figure to be placed within a larger figure.
```

Need to wordsmith this a bit.